### PR TITLE
Refactor : Call to Action

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
 - Trello task - https://trello.com/c/mSbMJ7X5/144-uag-block-wise-cleanup

### Types of changes
Breaking change

### How has this been tested?
 - Check Trello Card.

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by rohitp-bsf — APPROVED** [review]

File: src/blocks/call-to-action/render.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+						) }
+					</div>
+					{ ctaPosition == 'right' && isCta }
+				</div>
+			</div>
+		);
+	};
+
+	return (
+		<>
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/render.js:88]: Removed unwanted <>

File: src/blocks/call-to-action/save.js [hunk 2]
```diff
@@ -25,96 +21,107 @@ export default function save( props ) {
...
+						attributes={ props.attributes }
+						setAttributes="not_set"
+					/>
+				</div>
+			) }
+		</>
+	);
 
 	// Get Title components.
 	const title_text = (
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/save.js:45]: camelCase variable names.

File: src/blocks/call-to-action/settings.js [hunk 3]
```diff
@@ -0,0 +1,1176 @@
...
+import {
+	PanelBody,
+	SelectControl,
+	RangeControl,
+	TabPanel,
+	ToggleControl,
+	TextControl,
+} from '@wordpress/components';
+
+const svg_icons = Object.keys( UAGBIcon );
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/settings.js:34]: camelCase variable names.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Comment by RajkiranBagal** [comment]
Pull parent Branch.

File: src/blocks/call-to-action/render.js (hunk 1 above)
Comment (supriyajjadhav) [reply src/blocks/call-to-action/render.js]: Removed  <>

File: src/blocks/call-to-action/save.js (hunk 2 above)
Comment (supriyajjadhav) [reply src/blocks/call-to-action/save.js]: Updated

File: src/blocks/call-to-action/settings.js (hunk 3 above)
Comment (supriyajjadhav) [reply src/blocks/call-to-action/settings.js]: Updated

File: src/blocks/call-to-action/edit.js [hunk 4]
```diff
@@ -2,763 +2,53 @@
...
+			'uagb-cta-style-' + props.clientId.substr( 0, 8 )
+		);
+
+		if ( null !== element && undefined !== element ) {
+			element.innerHTML = CtaStyle( props );
+		}
+	}, [ props ] );
+
+	return (
+		<>
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/edit.js:45]: Remove unwanted <>

File: src/blocks/call-to-action/render.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+							</>
+						) }
+					</div>
+					{ ctaPosition == 'right' && isCta }
+				</div>
+			</div>
+		);
+	};
+
+	return (
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/render.js:87]: Add Suspense to whole return;

File: src/blocks/call-to-action/save.js [hunk 6]
```diff
@@ -25,96 +21,107 @@ export default function save( props ) {
...
-						     { title_text }
-						     { desc }
-						     { is_cta }
-						    </Fragment>
-						}
-
-						{ ( ctaPosition == "left" || ctaPosition == "right") &&
-							<Fragment>
-								{ title_text }
+		<>
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/save.js:59]: Remove unwanted <>

File: src/blocks/call-to-action/settings.js [hunk 7]
```diff
@@ -0,0 +1,1176 @@
...
+						setAttributes( { textAlign: value } )
+					}
+				/>
+			</BlockControls>
+		);
+	};
+
+	return (
+		<>
+			{ blockControls() }
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/settings.js:1160]: Add just one SUspense to whole return;

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**